parser_test.go

```diff
@@ -799,23 +872,55 @@ func TestParser_Parse(t *testing.T) {
 					{
 						Type: "comment",
 						Flags: map[string]Flag{
-							"comment": Flag{Values: []string{`kubernetes service traffic requiring SNAT`}},
+							"comment": {Values: []string{`kubernetes service traffic requiring SNAT`}},
 						},
 					},
 				},
 				Jump: &Target{
 					Name: "MASQUERADE",
 					Flags: map[string]Flag{
-						"random-fully": Flag{},
-						"random":       Flag{},
-						"to-ports": Flag{
+						"random-fully": {},
+						"random":       {},
+						"to-ports": {
 							Values: []string{"200-1000"},
 						},
 					},
 				},
 			},
 			err: nil,
 		},
+		{
+			name: "parse rule with match expression statistic",
+			s:    "-A foo ! -s 192.168.178.2 ! --dst 1.1.1.1 -m statistic --mode random --probability 0.50000000000 --every 10 --packet 5",
```

Comment:
hmm this is a bit funny. Now, rules that have a probability of 0.5 are different from rules with 0.50.

I wonder if we can do anything about that?